mps_youtube/main.py

```diff
@@ -3280,6 +3287,22 @@ def yt_url(url, print_title=0):
     if print_title:
         xprint(v.title)
 
+def yt_url_file(file_name):
+    """ Access a list of urls in a text file """
+
+    #Open and read the file
+    try:
+        with open(file_name, "r") as fo:
+            output = ','.join([line.strip('\n') for line in fo])
```

Comment:
File format would be more flexible with 
```py
[line.strip() for line in fo if line.strip()]
```
(allows for empty lines and strip all types of whitespaces)